File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

void printMarioPyramid(int);

int main(void)
{
    int height;
    do{
        height = get_int("Height: ");
    }while (!(height >= 1 && height <= 8));

    printMarioPyramid(height);
}

void printMarioPyramid(int h){
    for(int i = 0; i < h; i++)
    {
        for(int j = 0; j < h - (i + 1); j++)
        {
            printf(" ");
        }
        for(int k = 0; k < i + 1; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: pset6/mario/less/mario.py
from cs50 import get_int


def main():
    n = 0
    while True:
        n = get_int("Height: ")
        if not n < 1 and not n > 8:
            break
    print_sharp(n)


def print_sharp(n):
    counter = 1
    while n > 0:
        for i in range(n - 1):
            print(' ', end="")
        for i in range(counter):
            print('#', end="")
        counter += 1
        n -= 1
        print()

if __name__ == "__main__":
    main()
